.github/actions/run-cypress-tests/action.yml

```diff
@@ -56,14 +60,22 @@ runs:
       if: ${{ runner.os == 'Linux' }}
       run: |
         cd ./OpenSearch-Dashboards/plugins/security-dashboards-plugin
-        yarn runIdp
+        if [ -z "${{ inputs.osd_base_path }}" ]; then
+          yarn runIdp &
+        else
+          yarn runIdp --basePath ${{ inputs.osd_base_path }} &
+        fi
       shell: bash
 
     - name: Run OpenSearch Dashboards with provided configuration
       if: ${{ runner.os == 'Linux' }}
       run: |
         cd ./OpenSearch-Dashboards
-        nohup yarn start --no-base-path --no-watch --csp.warnLegacyBrowsers=false | tee dashboard.log &
+        if [ -z "${{ inputs.osd_base_path }}" ]; then
+          nohup yarn start --no-base-path --no-watch --csp.warnLegacyBrowsers=false | tee dashboard.log &
+        else
+          nohup yarn start --no-watch --csp.warnLegacyBrowsers=false | tee dashboard.log &
```

Comment:
Maybe I am missing something here, but how is this spinning up the OSD instance with the `inputs.osd_base_path`? Don't you need to specify it in the yml here? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml#L13

Reply:
Never mind I think it is coming from whoever is calling this action